jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/GradleProjectProperties.java

```diff
@@ -234,6 +234,8 @@ public JibContainerBuilder createJibContainerBuilder(
         case PACKAGED:
           // Add a JAR
           Jar jarTask = (Jar) project.getTasks().findByName("jar");
+          // TODO: in newer Gradle API (6.0?), maybe
+          // "Path jarPath = jarTask.getArchiveFile().get().getAsFile().toPath()" (untested).
```

Comment:
Maybe remove the TODO and just reopen this issue? https://github.com/GoogleContainerTools/jib/issues/2166